change/@fluentui-react-progress-d4feae76-06b5-4c51-ad92-246841f30aa4.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 1 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=28123&commitId=072167795e91c45baa58565ecb162c9ae4c2b3ec&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1943 screenshots unchanged, 0 screenshots with different dimensions and 1 screenshots with visible difference.


  <details><summary><b>unknown</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Field.ProgressBar.default.chromium.png|47| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>